docker-compose.full-demo.yml

```diff
@@ -0,0 +1,158 @@
+version: '3.2'
+
+# This docker-compose file sets up the services
+# and shared infrastructure on a common network called
+# `microservices_demo_1`. Because of this, all services
+# are able to communicate by their `container_name`.
+
+# Only the load balancer ports are mapped out. However, we
```

Comment:
"Only the load balancer ports need to be mapped out"  ?

Reply:
I would say "only the load balancer ports are exposed to the host"

Reply:
it's the "However, we also leave the ports for the databases and the rabbit queue
# mapped out" that confused me for a second. Those don't *need* to be exposed bu they are. The LB ports do need to be exposed for the deployment to be accessible.

Reply:
Yeah that's much better wording.  "exposed" didn't pop into my head. Changing...